tsconfig.json

```diff
@@ -15,7 +15,10 @@
     "include": [
         "typings/**/*.d.ts",
         "lib/**/*.ts",
+        "lib/apis/**/*.ts",
         "lib/bots/**/*.ts",
-        "lib/mixins/**/*.ts"
+        "lib/framework/**/*.ts",
+        "lib/services/**/*.ts",
+        "lib/utils/**/*.ts"
```

Comment:
I did actually have just that before, but I personally prefer being explicit. I can see the pros of both, tbh.